graphql/schema/types/metadata.graphql

```diff
@@ -1,15 +1,15 @@
 scalar Upload
 
 input GenerateMetadataInput {
-  sprites: Boolean!
-  previews: Boolean!
-  imagePreviews: Boolean!
+  sprites: Boolean
```

Comment:
You could also go with something like:

```
    sprites: Boolean! = False
```

This makes it optional on the client side, because there's a default value the server uses. And it avoids the trouble of having to check for nullability on the server side, while maintaining backwards compatibility.

The only problem with this approach is that the front-end clients need to understand this schema construction, and unfortunately, not all of them do.